src/plugins/oSnap/Proposal.vue

```diff
@@ -94,8 +94,9 @@ function enrichTransactionForDisplay(transaction: Transaction) {
 <template>
   <template v-if="safe.transactions.length > 0">
     <div
-      class="flex w-full flex-col gap-4 rounded-2xl border border-gray-200 p-3 md:p-4"
+      class="flex w-full flex-col gap-4 rounded-2xl border border-gray-200 p-3 md:p-4 relative"
     >
+      <OsnapMarketingWidget class="absolute top-[-16px] right-[16px]" />
```

Comment:
i think we might need logic to detect if the transactions are based on osnap

Reply:
this lives inside our plugin specific osnap proposal component so need for that.